### PR TITLE
FIx CI for NCCL plugins

### PR DESCRIPTION
**Pull request by Al3xR01**
/CC @avildema @artemry-nv @ilozgach

### CODE REVIEW

File: .ci/build_nccl_rdma_sharp_plugins.sh [hunk 1]
```diff
@@ -5,8 +5,9 @@ cd "${SCRIPT_DIR}"
 # shellcheck source=settings.sh
 . "${SCRIPT_DIR}/settings.sh"
 
-NCCL_RDMA_SHARP_PLUGINS_DIR="${NCCL_RDMA_SHARP_PLUGINS_DIR:-${WORKSPACE}/_install}"
```
Comment (Al3xR01) [inline .ci/build_nccl_rdma_sharp_plugins.sh:8]: moved to .ci/settings.sh

File: .ci/build_nccl_rdma_sharp_plugins.sh [hunk 2]
```diff
@@ -5,8 +5,9 @@ cd "${SCRIPT_DIR}"
 # shellcheck source=settings.sh
 . "${SCRIPT_DIR}/settings.sh"
```
Comment (Al3xR01) [inline .ci/build_nccl_rdma_sharp_plugins.sh:7]: moved to .ci/settings.sh

File: .ci/run_nccl_tests.sh [hunk 3]
```diff
@@ -149,7 +151,7 @@ do
 
                 if [ "${NCCL_ALGO}" = "CollNet" ]
                 then
```
Comment (Al3xR01) [inline .ci/run_nccl_tests.sh:153]: The option is no longer supported

File: .ci/run_nccl_tests.sh [hunk 4]
```diff
@@ -149,7 +151,7 @@ do
...
-                    MPIRUN_OPTIONS_NCCL_ALGO="-x NCCL_COLLNET_ENABLE=1 ${MPIRUN_OPTIONS_NCCL_ALGO}"
```
Comment (Al3xR01) [inline .ci/run_nccl_tests.sh:152]: The option is no longer supported

File: .ci/settings.sh [hunk 5]
```diff
@@ -1,39 +1,65 @@
...
 
 if [ "$DEBUG" = "true" ]
 then
     set -x
 fi
 
-export CUDA_VER="${CUDA_VER:-10.2}"
-echo "INFO: CUDA_VER = ${CUDA_VER}"
 
-module load ml/ci-tools
```
Comment (Al3xR01) [inline .ci/settings.sh:13]: Overrides HPCX_UBUNTU_INSTALL_DIR

**Comment by Al3xR01** [comment]
bot:mlx:test

**Comment by bureddy** [comment]
@Al3xR01 is this ready to merge?

**Comment by Al3xR01** [comment]
> @Al3xR01 is this ready to merge?

Positive

**Comment by bureddy** [comment]
bot:retest

**Pull request merged**